### PR TITLE
Update description for SQL references

### LINKED ISSUE #2594: Bug: abstract in "SQL functions and operators" is not helpful
### Where's the bug?

https://docs.risingwave.com/docs/next/sql-functions/

### Describe the bug

better to delete or update it

image

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

Some descriptions for SQL commands, query syntax, functions & operators, system catalogs are exactly the same as title, so modify them.

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2594

## Rendered preview

https://pr-2632.d2fbku9n2b6wde.amplifyapp.com/docs/next/query-syntax/

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/sql/commands/sql-create-sink.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: sql-create-sink
 title: CREATE SINK
-description: Create a sink.
+description: Deliver data processed in RisingWave to the sink.
```
Comment (xxchan) [inline docs/sql/commands/sql-create-sink.md]:
```suggestion
description: Deliver data processed in RisingWave to sinks.
```

File: docs/sql/commands/sql-set-background-ddl.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: sql-set-background-ddl
 title: SET BACKGROUND_DDL
-description: Run Data Definition Language (DDL) operations in the background.
+description: Run Data Definition Language operations in the background.
```
Comment (xxchan) [inline docs/sql/commands/sql-set-background-ddl.md]: I think DDL is more accurate, but maybe this is more user-friendly? 🤔
```suggestion
description: Run CREATE statements in the background.
```

File: docs/sql/commands/sql-set-time-zone.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 id: sql-set-time-zone
 title: SET TIME ZONE
-description: Set time zone.
+description: Specify time zone as needed.
```
Comment (xxchan) [inline docs/sql/commands/sql-set-time-zone.md]:
```suggestion
description: Specify time zone.
```

Comment (xxchan) [reply docs/sql/commands/sql-set-time-zone.md]: Or can we use sth like `description: null` ..?

**Review by xxchan — COMMENTED** [review]
"sql functions" page is not updated https://pr-2632.d2fbku9n2b6wde.amplifyapp.com/docs/next/sql-functions/ (which is origninal issue in https://github.com/risingwavelabs/risingwave-docs/issues/2594 

Changes of "sql syntax" generally LGTM

**Comment by WanYixian** [comment]
Updated, please take another look.

File: docs/sql/commands/sql-set-time-zone.md (hunk 3 above)
Comment (stdrc) [reply docs/sql/commands/sql-set-time-zone.md]: I don't think it's a big problem to have description the same as title.

File: docs/sql/query-syntax/query-syntax-generated-columns.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 id: query-syntax-generated-columns
 slug: /query-syntax-generated-columns
 title: Generated columns
+description: Generated from other columns in a table or source.
```
Comment (stdrc) [inline docs/sql/query-syntax/query-syntax-generated-columns.md]:
```suggestion
description: Generate columns from other columns in a table or source.
```

**Review by stdrc — APPROVED** [review]
Rest LGTM

**Review by st1page — APPROVED** [review]

**Pull request merged**